src/App.tsx

```diff
@@ -25,6 +28,7 @@ const App = () => {
                     <Route path="/auth" element={<AuthPage activeTab={'login'} />} />
                     {/* <Route path="/login" element={<LoginForm />} /> */}
                     <Route path="/parent-dashboard" element={<ParentDashboardPage />} />
+                    <Route path="/tutor-profile" element={<TutorProfilePage isUpdate />} />
```

Comment:
isUpdate  delete

Reply:
isUpdate flag is used in order to differentiate between update and create a new tutor.